aikau/src/test/resources/testApp/WEB-INF/classes/alfresco/site-webscripts/alfresco/renderers/CommentsList.get.js

```diff
@@ -19,6 +19,7 @@ model.jsonModel = {
          name: "alfresco/renderers/CommentsList",
          id: "COMMENT_LIST",
          config: {
+            addCommentsFullScreen: true,
```

Comment:
It's not the default, this is the test webscript. I will create another CommentsList on the page though, so that both are viewable. Not sure how easily this can be automatically tested? Would we just try and check how much space is around the dialog?